pkg/ds/daemon_set.go

```diff
@@ -45,6 +48,8 @@ type DaemonSet interface {
 	// Returns a list of all nodes that are selected by this daemon set's selector
 	EligibleNodes() ([]types.NodeName, error)
 
+	metricName(name string) string
```

Comment:
I actually don't think it is, according to `git grep \\bds\\.DaemonSet`, though it's less work to export the function. I don't really have a reason to keep it unexported... unless less work if anyone wants to implement it...?